spyder/plugins/editor/widgets/editor.py

```diff
@@ -857,36 +861,42 @@ def clone_from(self, other):
         self.set_stack_index(other.get_stack_index())
 
     @Slot()
-    def open_fileswitcher_dlg(self):
+    @Slot(str)
+    def open_switcher_dlg(self, initial_text=''):

```

Comment:
When is this method called? I did some tests and it seems that now we're always calling the MainWindow method to open the switcher, so perhaps this is not needed anymore?

Reply:
This method is called from the actions available on the Options Menu. When using them on a Editor window the Switcher is shown as part of the Editor window:

![image](https://user-images.githubusercontent.com/16781833/64536772-4cd3ce00-d2df-11e9-8d06-62ad0dde3153.png)

![image](https://user-images.githubusercontent.com/16781833/64536115-23667280-d2de-11e9-9628-41ef2d59d982.png)

Reply:
Ok, thanks for the clarification.